aquadoggo/src/db/stores/entry.rs

```diff
@@ -15,6 +15,20 @@ use p2panda_rs::storage_provider::traits::{AsStorageEntry, EntryStore};
 use crate::db::models::entry::EntryRow;
 use crate::db::provider::SqlStorage;
 
+// Re-write once https://github.com/pietgeursen/lipmaa-link/pull/3 merged in lipma-link
```

Comment:
This is merged and released in `0.2.1` :tada:

Reply:
And also, yeah it's annoying the fact that technically lipmaa links can go all the way back to 0 but the bamboo spec starts link numbers at 1.

:thinking:

Reply:
Yeh, I did a bit of a double take on that 😂

Reply:
I think I might add this to lipmaa link:

```rs
/// Get all the lipmaa link numbers back to some value
///
/// ```
/// use lipmaa_link::get_lipmaa_links_back_to;
///
/// let links = get_lipmaa_links_back_to(7, 1);
///
/// assert_eq!(links, vec![6, 5, 4, 1]);
///
/// ```
#[cfg(feature = "std")]
pub fn get_lipmaa_links_back_to(start: u64, lower_inclusive: u64) -> Vec<u64> {
    let mut path = Vec::new();

    let mut n = start;
    while n > lower_inclusive {
        n = lipmaa(n);
        path.push(n);
    }

    path
}
```

Reply:
Ok yeah, I think it's useful, published in `0.2.2`

Reply:
Nice! Thanks, I'll use that for sure.

Reply:
Oh yeah, I remember some one-off bug we had some time ago 😅